plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -497,6 +497,21 @@ USentryTransaction* SentrySubsystemDesktop::StartTransactionWithContextAndOption
 	return StartTransactionWithContext(context);
 }
 
+USentryTransactionContext* SentrySubsystemDesktop::ContinueTrace(const FString& sentryTrace, const TArray<FString>& baggageHeaders)
+{
+	sentry_transaction_context_t* nativeTransactionContext = sentry_transaction_context_new("<unlabeled transaction>", "default");
```

Comment:
"unlabeled transaction" is a default name set to every [transaction context](https://github.com/getsentry/sentry-java/blob/8586d1fe888254e57186cfaa74266703741da4cf/sentry/src/main/java/io/sentry/TransactionContext.java#L132) that's created via the `continueTrace` method in `sentry-java`. 

Since neither `sentry-native` nor `sentry-cocoa` provide similar API or recommendations on how to handle naming in case of trace propagation I've followed the Java approach here to unify the behavior across all platforms.